src/controllers/dashboardpage.js

```diff
@@ -12,27 +12,23 @@ define(["datetime", "events", "itemHelper", "serverNotifications", "dom", "globa
 
     function showPlaybackInfo(btn, session) {
         require(["alert"], function (alert) {
-            var showTranscodeReasons;
             var title;
             var text = [];
-            var displayPlayMethod = playMethodHelper.getDisplayPlayMethod(session);
-            var isDirectStream = "DirectStream" === displayPlayMethod;
-            var isTranscode = "Transcode" === displayPlayMethod;
 
-            if (isDirectStream) {
+            var displayPlayMethod = playMethodHelper.getDisplayPlayMethod(session);
+            if (displayPlayMethod === "DirectStream") {
                 title = globalize.translate("DirectStreaming");
                 text.push(globalize.translate("DirectStreamHelp1"));
                 text.push("<br/>");
                 text.push(globalize.translate("DirectStreamHelp2"));
-            } else if (isTranscode) {
+            } else if (displayPlayMethod === "Transcode") {
                 title = globalize.translate("Transcoding");
                 text.push(globalize.translate("MediaIsBeingConverted"));
-
                 if (session.TranscodingInfo && session.TranscodingInfo.TranscodeReasons && session.TranscodingInfo.TranscodeReasons.length) {
                     text.push("<br/>");
                     text.push(globalize.translate("LabelReasonForTranscoding"));
                     session.TranscodingInfo.TranscodeReasons.forEach(function (transcodeReason) {
-                        text.push(globalize.translate("" + transcodeReason));
```

Comment:
why change? this looks like a stringification safety net to me

Reply:
Javascript is usually very good about casting types and I didn't notice any issues when I was testing this pull request.